src/run/troubleshoot/analysis/AwsEnvironmentVariableAnalysis.ts

```diff
@@ -0,0 +1,92 @@
+ */
+export default class AwsEnvironmentVariableAnalysis extends BaseAnalysis {
+
+    async analyse ({ config }: TroubleshootAnalysisTaskContext): Promise<AnalysisOutcome> {
+        const profileIssues = this.checkAwsProfileEnvVariables();
+
+        const issues = profileIssues || this.checkAwsRegionEnvVariables();
+        const level = profileIssues ? "Fail" : "Warn";
+
+        return this.createOutcomeFrom(ANALYSIS_HEADLINE, issues, level);
+    }
+
+    /**
+     * Checks the AWS_PROFILE environment variable value
+     * Ensures the value is set and is one of the available profiles
+     * @returns { Promise<AnalysisIssue | undefined> } - Returns an issue if value is unset or not one of the available profile, otherwise undefined
+     */
+    private checkAwsProfileEnvVariables (): AnalysisIssue | undefined {
+        const AWS_PROFILE = process.env.AWS_PROFILE;
+        const _AWS_PROFILE = process.env._AWS_PROFILE;
+        const _DEFAULT_AWS_PROFILE = process.env._DEFAULT_AWS_PROFILE;
+        const profile = this.getAwsProfiles;
+
+        const isInvalidAwsProfile = (value:string | undefined):boolean => {
+            return !value || !profile.includes(value);
+        };
+
+        if (isInvalidAwsProfile(AWS_PROFILE) || isInvalidAwsProfile(_AWS_PROFILE) || isInvalidAwsProfile(_DEFAULT_AWS_PROFILE)) {
+            return this.createIssue(
+                "AWS_PROFILE environment variables are invalid or not set",
+                "Ensure the 'AWS_PROFILE', '_AWS_PROFILE', '_DEFAULT_AWS_PROFILE' environment variables are set to one of the following profiles: " + profile.join(", "),
+                SUGGESTIONS,
+                DOCUMENTATION_LINKS
+            );
+        }
+        return undefined;
+    }
+
+    /**
+     * Checks the _AWS_REGION environment variable value
+     * Ensures the value is set to the correct region
+     * @returns { Promise<AnalysisIssue | undefined> } - Returns an issue if value is unset or not the correct region, otherwise undefined
+     */
+    private checkAwsRegionEnvVariables (): AnalysisIssue | undefined {
+        const _AWS_REGION = process.env._AWS_REGION;
+        const _DEFAULT_AWS_REGION = process.env._DEFAULT_AWS_REGION;
+        const region = this.getAwsRegion;
+
+        const isInvalidAwsRegion = (value:string | undefined):boolean => {
+            return region !== value;
+        };
+
+        if (isInvalidAwsRegion(_AWS_REGION) || isInvalidAwsRegion(_DEFAULT_AWS_REGION)) {
+            return this.createIssue(
+                "AWS_REGION environment variables are invalid or not set",
+                "Ensure the '_AWS_REGION', '_DEFAULT_AWS_REGION' environment variables are set to region: " + region,
+                SUGGESTIONS,
+                DOCUMENTATION_LINKS
+            );
+        }
+        return undefined;
+
+    };
+
+    private get getAwsProfiles (): string[] {
```

Comment:
Either use properties or methods - as they could incur side effects probably best as methods. Naming a property `get...` could be confusing

```suggestion
    private getAwsProfiles (): string[] {
```